base/multi.jl

```diff
@@ -1029,8 +1029,14 @@ function serialize(s::AbstractSerializer, ex::CapturedException)
                                         # if a type used in ex.ex is undefined on the remote node.
 end
 
-original_ex(ex_str) = ErrorException(string("Error deserializing a CapturedException.",
-                                                " Original exception of type : ", ex_str))
+function original_ex(s::AbstractSerializer, ex_str, remote_stktrace)
+    local pid_str = ""
+    try pid_str = string(" from worker ", worker_id_from_socket(s.io)) end
```

Comment:
having `try ... end` all on the same line isn't good style for library code IMO, we often space out if, begin, for, etc for readability